src/index.html

```diff
@@ -5,8 +5,406 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="logo" class="header__image">
+      </a>
+
+      <nav class="header__nav">
+
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Смартфоны и телефоны  </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Гаджеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Планшеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Фото</a>
+          </li>
+          
+          <li class="header__item">
+            <a href="#" class="header__link">Видео</a>
+          </li>
+
+        </ul>
+      </nav>
+
+    </header>
+
+    <main class="content">
+      <div class="content__row">
+        <div class="content__card" data-qa="card">
+
+          <div class="content__image">
+            <img src="images/imac.jpeg" alt="card image" class="content__imac">
+          </div>
+    
+          <h2 class="content__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+    
+          <p class="content__code">
+            Код товара: 195434
+          </p>
+    
+          <div class="content__grup">
+            <div class="content__review">
+              <div class="content__stars">
```

Comment:
```suggestion
              <div class="content__stars content__stars--4">
```
Тут еще должен быть класс определяющий количество звезд

Reply:
Этот класс ты все еще не добавил